src/main/java/lotto/model/LottoTicket.java

```diff
@@ -0,0 +1,49 @@
+package lotto.model;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LottoTicket {
+
+    public static final Long PRICE = 1_000L;
+    public static final int SIZE = 6;
+    private final List<LottoNumber> lottoNumbers;
+
+    public LottoTicket(List<LottoNumber> lottoNumbers) {
+        validate(lottoNumbers);
+        this.lottoNumbers = lottoNumbers.stream()
+                .sorted(Comparator.comparing(LottoNumber::getNumber))
+                .collect(Collectors.toList());
+    }
+
+    private void validate(List<LottoNumber> lottoNumbers) {
+        validateSize(lottoNumbers);
+        validateDuplication(lottoNumbers);
+    }
+
+    private void validateDuplication(List<LottoNumber> lottoNumbers) {
+        if (lottoNumbers.stream().distinct().count() != SIZE) {
+            throw new IllegalArgumentException("로또 번호는 중복될 수 없습니다.");
+        }
+    }
+
+    private void validateSize(List<LottoNumber> lottoNumbers) {
+        if (lottoNumbers.size() != 6) {
+            throw new IllegalArgumentException("로또 번호는 6개만 가능합니다.");
+        }
+    }
+
+    public List<LottoNumber> getLottoNumbers() {
+        return this.lottoNumbers.stream()
+                .collect(Collectors.toUnmodifiableList());
```

Comment:
굳이 stream을 생성하지 않더라도 Colletions의 함수를 이용해서 방어적 복사가 가능해요! 😃
```suggestion
        return Collections.unmodifiableList(this.lottoNumbers);
```

Reply:
반영하였습니다.